addons/etcd/scripts/switchover-with-candidate.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+set -ex
+leaderEndpoint=${KB_LEADER_POD_FQDN}.${KB_NAMESPACE}.svc${CLUSTER_DOMAIN}:2379
```

Comment:
KB_LEADER_POD_FQDN is deprecated

Reply:
It is recommended to specify all node addresses in the etcd cluster using the --endpoints option when using the etcdctl command. This improves command reliability.